lisa/features/disks.py

```diff
@@ -79,20 +79,35 @@ def get_os_boot_partition(self) -> Optional[PartitionInfo]:
         for partition in partition_info:
             if partition.mount_point.startswith("/boot"):
                 boot_partition = partition
+                if "FreeBSD" in self._node.os.name:
```

Comment:
use `isinstance` to check if it's BSD. Only the os or some basic classes use the string to avoid cycle reference.

Reply:
Ok I will change it to use isinstance.